_source/feed-tweets.csv

```diff
@@ -0,0 +1,22 @@
+---
+layout: null
+---
+{%- assign twoYearsAgo = '11-05-2018 09:00:00' | date: '%s' -%}
+{%- assign tweetIndex = 2 -%}
+{%- capture linebreak %}
+{% endcapture -%}
+{%- assign posts = site.posts | reverse -%}
+{%- for post in posts -%}
+  {%- assign tags = post.tags | join: ' ' -%}
+  {%- capture postDate %}{{post.date | date: '%s'}}{% endcapture -%}
+{% if post.image and post.tweets and post.tweets[tweetIndex] and postDate > twoYearsAgo -%}
+  {%- capture dayAsSeconds %}{{ 86400 | times: forloop.index | minus: 12960000 }}{% endcapture -%}
+  {%- capture publishDate %}{{ site.time | date: '%s' | plus: dayAsSeconds }}{% endcapture -%}
```

Comment:
This math isn't great because `forloop.index` increments even when posts don't meat criteria, so there's spaces between publish dates.